extensions/proposals/WEBGL_compressed_texture_s3tc_srgb/extension.xml

```diff
@@ -0,0 +1,107 @@
+<?xml version="1.0"?>
+
+<proposal href="proposals/WEBGL_compressed_texture_s3tc_srgb/">
+  <name>WEBGL_compressed_texture_s3tc_srgb</name>
+  <contact>
+    <a href="https://www.khronos.org/webgl/public-mailing-list/">WebGL working group</a> (public_webgl 'at' khronos.org)
+  </contact>
+  <contributors>
+    <contributor>Members of the WebGL working group</contributor>
+  </contributors>
+  <number>1</number>
+  <depends>
+    <api version="1.0"/>
```

Comment:
I think it's fine if it's enabled independently because there isn't direct interactions.

Of course, for linear rendering we would enable EXT_sRGB, EXT_texture_compression_s3tc and WEBGL_compressed_texture_s3tc_srgb